style.css

```
@@ -0,0 +1,165 @@
+@charset "UTF-8";
+
+
+
+body{
+        
+    background-color:rgb(253, 251, 251) ;
+    
+    }
+#cabecalho{
+       
+    position: fixed;
+    color:rgba(69, 161, 231, 0.246) ;         
+    font-family: 'Lato';
+    top: 0px;
+    background-color:rgba(119, 12, 48, 0.8);
+    z-index: 2;
+
+    }
+
+img{
+    position: absolute;
+    display: block;
+    top: 430px;
+    left:-150px;
+    size: 60%;
+    border: 1px;
+    border-color: black;
+    padding:1px;
+    width: 500px;
+    
+
+    }
+
+
+#cabecalho{
+
+    width: 1200px;
+
+    }
+
+
+#titulo{
+    font-size: 60px;
+    margin: 8px;
+    font-family: 'Dancing Script', cursive;
+    padding-left: 40px;
+    padding-top: 15px;
+    font-style: italic;
+    z-index:0;
+    color:white;
+    }
+
+
+#cabecabutons li {
+
+    display: inline-block;
+    margin: 8px;
+        
+     }
+
+
+ul li a{
+            
+    color:rgb(248, 221, 218);
+    font-size: 25px;
+    padding: 8px;
+    }
+    
+ul li a:hover {
+
+    background-color:rgba (9, 106, 182, 0.5);
+    transition-delay: 100ms;
+    transition-duration: 1s;
+    
+    }
+    
+    
+aside {
+      
+    
+    font-family: 'lato';
+    font-style: italic;
+      
+    
+
+    }
+
+      
+#rodape{
+    
+    background-color: rgb(169, 192, 236);     
+    
+          
+    }
+/* aqui é a formatação do texto maior*/
+.textodentro{
+    
+    font-family: 'lato';
+    text-align: justify;
+    background-color: rgb(69, 200, 240);
+    width: 400px;
+    margin-top: 250px;
+    margin-left: 200px;
+    padding: 10px;
+
+    }
+
+    #sectando {
+        text-align: justify;
+        background-color: rgb(69, 200, 240);
+        width: 300px;
+        margin-top: -400px;
+        margin-left: 199px;
+        padding: 10px;
+        
+    }
+
+
+    .listalateral{
+        margin-top: 60px;
+        margin-left: 1040px;
+    }
+
+    
+    footer{
+    background-color: rgba(59, 138, 241, 0.486);
+    width: 1200px;
+    padding: 20px;
+    font-size: 25px;
+    margin-top: 60%;
+     
+
+        
+    }
+
+    footer address{
+        font-weight:bold ;
+        color: lightslategray;
+        text-align: center;
+    }
+
+    #rodape h1{
+        font-weight:bold ;
+        color: lightslategray;
+        text-align: center;
+        font-size:25px;
+    }
+
+
+
+    table {
+        font-family: 'lato', sans-serif;
+        border-collapse: collapse;
+        width: 20%;
+      }
+      
+      td, th {
+        border: 1px solid #110f0f;
+        text-align: left;
+        padding: 8px;
+        
+      }
+
+      
```

Sempre uma linha vazia depois da última linha de código! 

```suggestion
      }

```